src/translations/screens/Profile.ts

```diff
@@ -88,6 +88,13 @@ const ProfileTexts = {
             'Activate to edit favourite departures',
           ),
         },
+        frontpageFavourites: {
+          label: _('Avganger på forside', 'Dashboard departures'),
```

Comment:
Should this be `Departures on the front page` instead?

Reply:
We're moving it to developer options, so don't think I will spend time changing it, but it might be more user-friendly for end users :)